catroid/src/main/java/org/catrobat/catroid/formulaeditor/InternFormulaKeyboardAdapter.java

```diff
@@ -480,6 +485,21 @@
 		return returnList;
 	}
 
+	private List<InternToken> buildTripleParameterFunction(Functions function, InternTokenType firstParameter,
+			String firstParameterNumberValue, InternTokenType secondParameter, String secondParameterNumberValue,
+			InternTokenType thirdParameter, String thirdParameterNumberValue) {
+		List<InternToken> returnList = new LinkedList<InternToken>();
+		returnList.add(new InternToken(InternTokenType.FUNCTION_NAME, function.name()));
```

Comment:
InterTokenType.x is already imported. Please use "FUNCTION_NAME" without "InternTokeType." like in buildDoubleParameterFunction(). Also apply this change to each used InternTokeType in buildTripleParameterFunction().